static/src/stylesheets/_common.scss

```diff
@@ -19,6 +19,7 @@
 @import 'module/commercial/_commercial';
 @import 'module/crosswords/_main';
 @import 'module/email/_main';
+@import 'module/signup/_main';
```

Comment:
I thought we split out the CSS for these pages when it was survey? Can we do that for signup?